internal/release/RELEASE-NOTES

```diff
@@ -1,7 +1,6 @@
-This is the v0 release of the 1Password Go SDK.
```

Comment:
note for reviewer - ignore the changes in this file, we're deciding about the content here separately.